server.js

```diff
@@ -2195,6 +2195,49 @@ cache(function(data, match, sendBadge, request) {
         badgeData.text[1] = versions.sort().join(', ');
         badgeData.colorscheme = 'blue';
         sendBadge(format, badgeData);
+      } else if (info === 'djversions') {
+        let versions = [];
+        let pattern = /^Framework :: Django :: ([\d.]+)$/;
+        for (let i = 0; i < parsedData.info.classifiers.length; i++) {
+          let matched = pattern.exec(parsedData.info.classifiers[i]);
+          if (matched && matched[1]) {
+            versions.push(matched[1]);
+          }
+        }
+        if (!versions.length) {
+          versions.push('not found');
+        }
+
```

Comment:
Could you extract some the version parsing and formatting into a helper function in `lib/pypi-helpers.js`?